__tests__/app.test.js

```diff
@@ -0,0 +1,47 @@
+const app = require('../app.js');
+const db = require('../db/connection.js')
+const seed = require('../db/seeds/seed.js');
+const testData = require('../db/data/test-data/index.js');
+const request = require('supertest');
+
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe('GET /api/topics', () => {
+    it('responds with 200 status', () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+    })
+    it('responds with an array of topic objects with length 3', () => {
+        return request(app)
+        .get("/api/topics")
+        .then((res) => {
+            const topics = res.body.topics;
+            expect(topics).toHaveLength(3);
```

Comment:
This expect line can be made a part of the next 'it' block and save a bit of bulk on this test suite